*** source/aimk.orig	2006-04-11 13:58:41.000000000 -0500
--- source/aimk	2006-04-11 13:59:36.000000000 -0500
***************
*** 1115,1124 ****
     set REL = `/usr/bin/uname -r`
     switch ($REL)
     case "6.*":
!       set CFLAGS = "$OFLAG -no-cpp-precomp -flat_namespace -Wall -Werror -Wstrict-prototypes -DFD_SETSIZE=8192 -DDARWIN -DDARWIN6 -DGETHOSTBYNAME -DGETHOSTBYADDR_M $DEBUG_FLAG $CFLAGS"
        breaksw
     case "7.*":
!       set CFLAGS = "$OFLAG -no-cpp-precomp -flat_namespace -Wall -Werror -Wstrict-prototypes -DFD_SETSIZE=8192 -DDARWIN -DDARWIN7 -DGETHOSTBYNAME -DGETHOSTBYADDR_M $DEBUG_FLAG $CFLAGS"
        breaksw
     default:
        echo DARWIN release \"$REL\" not supported!
--- 1115,1124 ----
     set REL = `/usr/bin/uname -r`
     switch ($REL)
     case "6.*":
!       set CFLAGS = "$OFLAG -no-cpp-precomp -flat_namespace -Wall -Wstrict-prototypes -DFD_SETSIZE=8192 -DDARWIN -DDARWIN6 -DGETHOSTBYNAME -DGETHOSTBYADDR_M $DEBUG_FLAG $CFLAGS"
        breaksw
     case "7.*":
!       set CFLAGS = "$OFLAG -no-cpp-precomp -flat_namespace -Wall -Wstrict-prototypes -DFD_SETSIZE=8192 -DDARWIN -DDARWIN7 -DGETHOSTBYNAME -DGETHOSTBYADDR_M $DEBUG_FLAG $CFLAGS"
        breaksw
     default:
        echo DARWIN release \"$REL\" not supported!
***************
*** 1252,1258 ****
     set COMPILE_DC = 1
     set MAKE = gmake
  
!    set CFLAGS   = "$OFLAG -Werror -woff 3201,3496 -n32 -mips3 -mp -DFD_SETSIZE=8192 -DIRIX -D$BUILDARCH -DGETHOSTBYNAME_M -DGETHOSTBYADDR_M $DEBUG_FLAG $CFLAGS"
     set NOERR_CFLAG = "-woff 1110,1174,1209"
     set DEPEND_FLAGS = "$CFLAGS $XMTINCD"
     set LFLAGS   = "-n32 -mips3 $DEBUG_FLAG $LFLAGS"
--- 1252,1258 ----
     set COMPILE_DC = 1
     set MAKE = gmake
  
!    set CFLAGS   = "$OFLAG -woff 3201,3496 -n32 -mips3 -mp -DFD_SETSIZE=8192 -DIRIX -D$BUILDARCH -DGETHOSTBYNAME_M -DGETHOSTBYADDR_M $DEBUG_FLAG $CFLAGS"
     set NOERR_CFLAG = "-woff 1110,1174,1209"
     set DEPEND_FLAGS = "$CFLAGS $XMTINCD"
     set LFLAGS   = "-n32 -mips3 $DEBUG_FLAG $LFLAGS"
***************
*** 1385,1391 ****
        endif
     endif
  
!    set CFLAGS = "$OFLAG -Wall -Werror -Wstrict-prototypes -D__GRIDENGINE_FD_SETSIZE=8192 -DLINUX -D$BUILDARCH -D$COMPILE_ARCH -D_GNU_SOURCE -DGETHOSTBYNAME_R6 -DGETHOSTBYADDR_R8 $DEBUG_FLAG $CFLAGS"
     if ("$ADOC" == 1) then
        set CFLAGS = "$CFLAGS -Wno-error"
     endif
--- 1385,1391 ----
        endif
     endif
  
!    set CFLAGS = "$OFLAG -Wall -Wstrict-prototypes -D__GRIDENGINE_FD_SETSIZE=8192 -DLINUX -D$BUILDARCH -D$COMPILE_ARCH -D_GNU_SOURCE -DGETHOSTBYNAME_R6 -DGETHOSTBYADDR_R8 $DEBUG_FLAG $CFLAGS"
     if ("$ADOC" == 1) then
        set CFLAGS = "$CFLAGS -Wno-error"
     endif
***************
*** 1483,1489 ****
        set CFLAGS = "$DEBUG_FLAG $CFLAGS"
        set CFLAGS = "-DGETHOSTBYADDR_M -D_ALL_SOURCE -D_REENTRANT $CFLAGS"
        set CFLAGS = "-DINTERIX -D$COMPILE_ARCH -DGETHOSTBYNAME_M $CFLAGS" 
!       set CFLAGS = "-Wall -Werror -O0 -pipe $CFLAGS"
        #set LFLAGS = "-Wl,-EL,-b=pe-i1386 $LFLAGS"
     else
        set CC  = cc
--- 1483,1489 ----
        set CFLAGS = "$DEBUG_FLAG $CFLAGS"
        set CFLAGS = "-DGETHOSTBYADDR_M -D_ALL_SOURCE -D_REENTRANT $CFLAGS"
        set CFLAGS = "-DINTERIX -D$COMPILE_ARCH -DGETHOSTBYNAME_M $CFLAGS" 
!       set CFLAGS = "-Wall -O0 -pipe $CFLAGS"
        #set LFLAGS = "-Wl,-EL,-b=pe-i1386 $LFLAGS"
     else
        set CC  = cc
***************
*** 1602,1608 ****
        set DEBUG_FLAG = "$DEBUG_FLAG -pg -DSGE_LOCK_DEBUG"
     endif
  
!    set CFLAGS   = "$OFLAG -Wall -Werror -Wstrict-prototypes -DLINUX -DSLINUX -DGETHOSTBYNAME_R6 -DGETHOSTBYADDR_R8 $DEBUG_FLAG $CFLAGS"
     set DEPEND_FLAGS = "$CFLAGS $XMTINCD"
     set LFLAGS   = "$DEBUG_FLAG $LFLAGS"
  
--- 1602,1608 ----
        set DEBUG_FLAG = "$DEBUG_FLAG -pg -DSGE_LOCK_DEBUG"
     endif
  
!    set CFLAGS   = "$OFLAG -Wall -Wstrict-prototypes -DLINUX -DSLINUX -DGETHOSTBYNAME_R6 -DGETHOSTBYADDR_R8 $DEBUG_FLAG $CFLAGS"
     set DEPEND_FLAGS = "$CFLAGS $XMTINCD"
     set LFLAGS   = "$DEBUG_FLAG $LFLAGS"
  
***************
*** 1706,1712 ****
        endif
        set CFLAGS = "-Xc -v $OFLAG $MEMFLAG -ftrap=division $CFLAGS $DEBUG_FLAG"
     else if ( $CC == gcc) then
!       #set CFLAGS = "-Wall -Werror -Wstrict-prototypes $CFLAGS"
        set CFLAGS = "-Wall -Wstrict-prototypes $CFLAGS"
        set SHARED_CFLAGS = "-fPIC"
        set SHARED_LFLAGS = "$SHARED_LFLAGS -L. -shared"
--- 1706,1712 ----
        endif
        set CFLAGS = "-Xc -v $OFLAG $MEMFLAG -ftrap=division $CFLAGS $DEBUG_FLAG"
     else if ( $CC == gcc) then
!       #set CFLAGS = "-Wall -Wstrict-prototypes $CFLAGS"
        set CFLAGS = "-Wall -Wstrict-prototypes $CFLAGS"
        set SHARED_CFLAGS = "-fPIC"
        set SHARED_LFLAGS = "$SHARED_LFLAGS -L. -shared"
