--- pyslurm-15.08.2/setup.py	2016-03-04 10:27:19.000000000 +0000
+++ pyslurm-15.08.2.new/setup.py	2016-03-04 10:34:28.224216360 +0000
@@ -24,7 +24,7 @@
 #VERSION = imp.load_source("/tmp", "pyslurm/__init__.py").__version__
 __version__ = "15.08.2"
 __min_slurm_hex_version__ = "0x0f0802"
-__max_slurm_hex_version__ = "0x0f0803"
+__max_slurm_hex_version__ = "0x0f0808"
 
 def fatal(logstring, code=1):
 	logger.error("Fatal: " + logstring)
