diff -ru pyslurm-17.11.0.7.old/doc/source/conf.py pyslurm-17.11.0.7/doc/source/conf.py
--- pyslurm-17.11.0.7.old/doc/source/conf.py	2018-04-20 11:22:40.984128538 +0000
+++ pyslurm-17.11.0.7/doc/source/conf.py	2018-04-20 11:41:36.514740469 +0000
@@ -100,7 +100,7 @@
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
-html_theme = 'classic'
+html_theme = 'default'
 
 # Theme options are theme-specific and customize the look and feel of a theme
 # further.  For a list of options available for each theme, see the
