--- etc/global/bashrc.in.orig	2012-06-15 17:16:11.286954716 +0200
+++ etc/global/bashrc.in	2012-06-15 17:16:22.124017398 +0200
@@ -11,7 +11,7 @@
        -zsh|zsh|*/zsh)	modules_shell=zsh ;;
     -bash|bash|*/bash)	modules_shell=bash ;;
 esac
-@VERSIONING@eval "module() { eval \`@BASEPREFIX@/Modules\$MODULE_VERSION/bin/modulecmd $modules_shell \$*\`; }"
+@VERSIONING@eval "module() { eval \`@BASEPREFIX@/Modules/\$MODULE_VERSION/bin/modulecmd $modules_shell \$*\`; }"
 @NOTVERSIONING@eval "module() { eval \`@bindir@/modulecmd $modules_shell \$*\`; }"
 
 unset modules_shell
