--- oscar_table.sql	2013-01-31 16:10:44.000000000 -0500
+++ oscar_table.sql.oldmysql	2013-01-31 16:31:43.000000000 -0500
@@ -45,7 +45,7 @@
     PRIMARY KEY (id, parent_id),
     KEY parent_id ( parent_id ),
     CONSTRAINT Clusters_ibfk_1 FOREIGN KEY (parent_id) REFERENCES Clusters (id) ON DELETE CASCADE
-)ENGINE=INNODB DEFAULT CHARSET=utf8;
+)TYPE=INNODB;
 
 -- Groups
 CREATE TABLE IF NOT EXISTS Groups(
@@ -53,20 +53,20 @@
     name VARCHAR(100)  not null unique,
     selected  integer  DEFAULT '0',
     type VARCHAR(100)
-)ENGINE=INNODB DEFAULT CHARSET=utf8;
+)TYPE=INNODB;
 
 -- Status
 CREATE TABLE IF NOT EXISTS Status(
     id  integer auto_increment not null unique primary key,
     name VARCHAR(100)  not null unique
-)ENGINE=INNODB DEFAULT CHARSET=utf8;
+)TYPE=INNODB;
 
 -- Package_status
 
 CREATE TABLE IF NOT EXISTS Package_status(
 	id integer auto_increment not null unique primary key,
 	status VARCHAR(50)
-)ENGINE=INNODB DEFAULT CHARSET=utf8;
+)TYPE=INNODB;
 
 -- Packages
 CREATE TABLE IF NOT EXISTS Packages(
@@ -85,7 +85,7 @@
     vendor VARCHAR(100),
     description  text,
     KEY package ( package )
-)ENGINE=INNODB DEFAULT CHARSET=utf8;
+)TYPE=INNODB;
 
 -- Images
 CREATE TABLE IF NOT EXISTS Images(
@@ -93,7 +93,7 @@
     id  integer   auto_increment not null unique primary key,
     name VARCHAR(100)  not null unique,
     path VARCHAR(100)
-)ENGINE=INNODB DEFAULT CHARSET=utf8;
+)TYPE=INNODB;
 
 -- Nodes
 CREATE TABLE IF NOT EXISTS Nodes(
@@ -117,7 +117,7 @@
     KEY group_name ( group_name ),
     CONSTRAINT Nodes_ibfk_1 FOREIGN KEY (cluster_id) REFERENCES Clusters (id) ON DELETE CASCADE,
     CONSTRAINT Nodes_ibfk_2 FOREIGN KEY (group_name) REFERENCES Groups (name) ON DELETE CASCADE ON UPDATE CASCADE
-)ENGINE=INNODB DEFAULT CHARSET=utf8;
+)TYPE=INNODB;
 
 -- OscarFileServer
 CREATE TABLE IF NOT EXISTS OscarFileServer(
@@ -139,7 +139,7 @@
     rfc1918 VARCHAR(100),
     KEY cluster_id ( cluster_id ),
     CONSTRAINT Networks_ibfk_1 FOREIGN KEY (cluster_id) REFERENCES Clusters (id) ON DELETE CASCADE
-)ENGINE=INNODB DEFAULT CHARSET=utf8;
+)TYPE=INNODB;
 
 -- Nics
 CREATE TABLE IF NOT EXISTS Nics(
@@ -153,7 +153,7 @@
     KEY network_id ( network_id ),
     CONSTRAINT Nics_ibfk_1 FOREIGN KEY (node_id) REFERENCES Nodes (id) ON DELETE CASCADE,
     CONSTRAINT Nics_ibfk_2 FOREIGN KEY (network_id) REFERENCES Networks (n_id) ON DELETE CASCADE
-)ENGINE=INNODB DEFAULT CHARSET=utf8;
+)TYPE=INNODB;
 
 -- Packages_servicelists
 
@@ -166,7 +166,7 @@
     KEY group_name ( group_name ),
     CONSTRAINT Packages_servicelists_ibfk_1 FOREIGN KEY (package) REFERENCES Packages (package) ON DELETE CASCADE ON UPDATE CASCADE,
     CONSTRAINT Packages_servicelists_ibfk_2 FOREIGN KEY (group_name) REFERENCES Groups (name) ON DELETE CASCADE ON UPDATE CASCADE
-)ENGINE=INNODB DEFAULT CHARSET=utf8;
+)TYPE=INNODB;
 
 -- Packages_switcher
 CREATE TABLE IF NOT EXISTS Packages_switcher(
@@ -176,7 +176,7 @@
     PRIMARY KEY (package, switcher_name),
 --    KEY package ( package ),
     CONSTRAINT Packages_switcher_ibfk_1 FOREIGN KEY (package) REFERENCES Packages (package) ON DELETE CASCADE ON UPDATE CASCADE
-)ENGINE=INNODB DEFAULT CHARSET=utf8;
+)TYPE=INNODB;
 
 -- Packages_config
 CREATE TABLE IF NOT EXISTS Packages_config(
@@ -188,7 +188,7 @@
     PRIMARY KEY (config_id, package),
     KEY package ( package ),
     CONSTRAINT Packages_config_ibfk_1 FOREIGN KEY (package) REFERENCES Packages (package) ON DELETE CASCADE
-)ENGINE=INNODB DEFAULT CHARSET=utf8;
+)TYPE=INNODB;
 
 -- Node_Package_Status
 CREATE TABLE IF NOT EXISTS Node_Package_Status(
@@ -214,7 +214,7 @@
     CONSTRAINT Node_Package_Status_ibfk_3 FOREIGN KEY (requested) REFERENCES Status (id) ON DELETE CASCADE,
     CONSTRAINT Node_Package_Status_ibfk_4 FOREIGN KEY (curr) REFERENCES Status (id) ON DELETE CASCADE,
     CONSTRAINT Node_Package_Status_ibfk_5 FOREIGN KEY (status) REFERENCES Package_status (id) ON DELETE CASCADE
-)ENGINE=INNODB DEFAULT CHARSET=utf8;
+)TYPE=INNODB;
 
 -- Group_Nodes
 CREATE TABLE IF NOT EXISTS Group_Nodes(
@@ -225,7 +225,7 @@
     KEY group_name ( group_name ),
     CONSTRAINT Group_Nodes_ibfk_1 FOREIGN KEY (node_id) REFERENCES Nodes (id) ON DELETE CASCADE,
     CONSTRAINT Group_Nodes_ibfk_2 FOREIGN KEY (group_name) REFERENCES Groups (name) ON DELETE CASCADE ON UPDATE CASCADE
-)ENGINE=INNODB DEFAULT CHARSET=utf8;
+)TYPE=INNODB;
 
 -- Group_Packages
 -- This table is used to saved information from Selector, i.e., information
@@ -239,7 +239,7 @@
     KEY group_name ( group_name ),
     CONSTRAINT Group_Packages_ibfk_1 FOREIGN KEY (package) REFERENCES Packages (package) ON DELETE CASCADE,
     CONSTRAINT Group_Packages_ibfk_2 FOREIGN KEY (group_name) REFERENCES Groups (name) ON DELETE CASCADE ON UPDATE CASCADE
-)ENGINE=INNODB DEFAULT CHARSET=utf8;
+)TYPE=INNODB;
 
 -- Image_Package_Status
 CREATE TABLE IF NOT EXISTS Image_Package_Status(
@@ -265,7 +265,7 @@
     CONSTRAINT Image_Package_Status_ibfk_3 FOREIGN KEY (requested) REFERENCES Status (id) ON DELETE CASCADE,
     CONSTRAINT Image_Package_Status_ibfk_4 FOREIGN KEY (curr) REFERENCES Status (id) ON DELETE CASCADE,
     CONSTRAINT Image_Package_Status_ibfk_5 FOREIGN KEY (status) REFERENCES Package_status (id) ON DELETE CASCADE
-)ENGINE=INNODB DEFAULT CHARSET=utf8;
+)TYPE=INNODB;
 
 -- Wizard_status
 CREATE TABLE IF NOT EXISTS Wizard_status(
@@ -287,7 +287,7 @@
     partition_id      integer   auto_increment not null unique primary key,
     name              CHAR(50),
     distro            CHAR(50)
-)ENGINE=INNODB DEFAULT CHARSET=utf8;
+)TYPE=INNODB;
 
 -- Cluster_Partitions
 CREATE TABLE IF NOT EXISTS Cluster_Partitions(
@@ -298,7 +298,7 @@
     KEY             partition_id    ( partition_id ),
     CONSTRAINT      Cluster_Partitions_ibfk_1 FOREIGN KEY (cluster_id) REFERENCES Clusters (id) ON DELETE CASCADE,
     CONSTRAINT      Cluster_Partitions_ibfk_2 FOREIGN KEY (partition_id) REFERENCES Partitions (partition_id) ON DELETE CASCADE
-)ENGINE=INNODB DEFAULT CHARSET=utf8;
+)TYPE=INNODB;
 
 -- Partition_Nodes
 CREATE TABLE IF NOT EXISTS Partition_Nodes(
@@ -310,7 +310,7 @@
     KEY             node_id         ( node_id ),
     CONSTRAINT      Partition_Nodes_ibfk_1 FOREIGN KEY (partition_id) REFERENCES Partitions (partition_id) ON DELETE CASCADE,
     CONSTRAINT      Partition_Nodes_ibfk_2 FOREIGN KEY (node_id) REFERENCES Nodes (id) ON DELETE CASCADE
-)ENGINE=INNODB DEFAULT CHARSET=utf8;
+)TYPE=INNODB;
 
 INSERT INTO Wizard_status VALUES(0,'download_packages','');
 INSERT INTO Wizard_status VALUES(1,'select_packages','');
